src/System.Private.CoreLib/src/System/Runtime/GCSettings.CoreCLR.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime
+{
+    public static partial class GCSettings
+    {
+        public static bool IsServerGC =>
```

Comment:
Nit: The FCalls can be moved from `System.GC` to here to avoid extra level of forwarders:

```
public static extern bool IsServerGC
{
    [MethodImplAttribute(MethodImplOptions.InternalCall)]
    get;
}

[MethodImplAttribute(MethodImplOptions.InternalCall)]
private static extern GCLatencyMode GetLatencyMode();

...
```

Reply:
Thanks. Done.